lib/Config/Menu.php

```diff
@@ -439,15 +446,7 @@ function migration() {
   }
 
   function welcomeWizard() {
-    if ((bool)(defined('DOING_AJAX') && DOING_AJAX)) return;
-    $data = [
-      'is_mp2_migration_complete' => (bool)$this->settings->get(MP2Migrator::MIGRATION_COMPLETE_SETTING_KEY),
-      'is_woocommerce_active' => $this->woocommerce_helper->isWooCommerceActive(),
-      'finish_wizard_url' => $this->wp->adminUrl('admin.php?page=' . self::MAIN_PAGE_SLUG),
-      'sender' => $this->settings->get('sender'),
-      'admin_email' => get_option('admin_email'),
-    ];
-    $this->page_renderer->displayPage('welcome_wizard.html', $data);
+    $this->container->get(WelcomeWizard::class)->render();
```

Comment:
You're fetching this service from DI container but the service is not public. That generates warnings. Probably not a big issue but still... Thinking about it, I think we should just mark all services as public (automatically maybe) because we'll just always need this (also like fetching services in integration tests, etc.).

Reply:
Now I see another commit fixed that.